test/testapps/Whitespace in path/Models/ErrorViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Whitespace_in_path.Models
```

Comment:
This app is just a copy-paste and rename of `SimpleMvc`.

Reply:
We could consider modifying `ProjectDirectory` to accept a target path and deploy SimpleMvc to a path with spaces. Might let us test a variant where the project name itself doesn't have spaces but the path does e.g. `%temp%\razor\path with spaces\SimpleMvc`